benchmark/pounce_benchmark.py

```diff
@@ -42,8 +42,9 @@ def benchmark_coreset_algorithms(
     """
     Benchmark coreset algorithms by processing a video GIF.
 
-    :param in_path: Path to the input GIF file.
-    :param out_dir: Directory to save the output GIFs for each coreset algorithm.
+    :param in_path: Path to the input GIF file, relative to the script's location.
+    :param out_dir: Directory to save the output GIFs for each coreset algorithm,
+                    relative to the script's location.
```

Comment:
Please use a single indent on any continuation line. If you try to align to the start of the line above, something inevitably ends up misaligned when something is renamed later.